src/WebJobs.Script/FileTraceWriter.cs

```diff
@@ -38,9 +40,17 @@ public FileTraceWriter(string logFilePath, TraceLevel level) : base(level)
             }
             else
             {
```

Comment:
Not part of your change, but looking at this class, looks like we're missing a call to the base type's Dispose(bool) method.